lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -9,7 +9,11 @@ def initialize(engine_config:, directory:)
 
         def run
           files.map do |file|
-            process_file(file)
+            begin
+              process_file(file)
+            rescue => e
+              $stderr.puts e
```

Comment:
I'm not sure what this will look like. I might get a touch fancier / more explicit:

```rb
$stderr.puts "Skipping file #{file} due to exception"
$stderr.puts "(#{ex.class}) #{ex.message} #{ex.backtrace.join("\n")}"
```